src/components/views/rooms/SendMessageComposer.tsx

```diff
@@ -383,7 +384,7 @@ export class SendMessageComposer extends React.Component<ISendMessageComposerPro
         if (!containsEmote(model) && isSlashCommand(this.model)) {
             const [cmd, args, commandText] = getSlashCommand(this.model);
             if (cmd) {
-                const threadId = this.props.relation?.rel_type === RelationType.Thread
+                const threadId = this.props.relation?.rel_type === THREAD_RELATION_TYPE.name
```

Comment:
Shouldn't this use `THREAD_RELATION_TYPE.matches(relation?.rel_type)` to support both legacy and new?